nucleus-api/src/main/java/io/github/nucleuspowered/nucleus/api/query/QueryComparator.java

```diff
@@ -0,0 +1,35 @@
+package io.github.nucleuspowered.nucleus.api.query;
+
+public enum QueryComparator {
+
+    /**
+     * General comparators.
+     */
+    LESS_THAN("<", 1),
+    LESS_THAN_OR_EQUAL_TO("<=", 1),
+    EQUALS("=", 1),
+    GREATER_THAN_OR_EQUAL_TO(">=", 1),
+    GREATER_THAN(">", 1),
+
+    /**
+     * Compares if a timestamp is between a certain range of timestamps.
+     */
+    BETWEEN("BETWEEN", 2);
```

Comment:
I've updated columns to use another system, if you like it I can update QueryComparator to use a similar one.